lib/cli/src/store.rs

```diff
@@ -331,9 +331,8 @@ impl StoreOptions {
 // If we don't have a compiler, but we have an engine
 #[cfg(not(feature = "compiler"))]
 impl StoreOptions {
-    fn get_engine_headless(&self) -> Result<Arc<dyn Engine + Send + Sync>> {
-        let engine: Arc<dyn Engine + Send + Sync> =
-            Arc::new(wasmer_compiler::Universal::headless().engine());
+    fn get_engine_headless(&self) -> Result<Arc<Engine>> {
+        let engine: Arc<Engine> = Arc::new(wasmer_compiler::Universal::headless().engine());
```

Comment:
Why is this named Universal? It should be named `Engine` ?

```suggestion
        let engine: Arc<Engine> = Arc::new(wasmer_compiler::Engine::headless());
```